src/app/progress/milestone-items.ts

```diff
@@ -385,7 +385,9 @@ export function recordToPursuitItem(
   };
 
   if (record.recordDef.rewardItems) {
-    dimItem.pursuit.rewards = record.recordDef.rewardItems;
+    dimItem.pursuit.rewards = record.recordDef.rewardItems.filter(
+      (_r, i) => record.recordComponent.rewardVisibilty?.[i] ?? true,
+    );
```

Comment:
This is fun, turns out there was a way to tell which rewards are actually visible!